.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+## Issue that this pull request solves
+
+Closes: # (issue number)
+
+## Proposed changes
+
+Brief description of what is fixed or changed
+
+- Files changed
+- Dependencies if any (on other issues/PRs)
+- Basic tests done to validate
+- Conflicts if any (describe the reason for conflict)
+
+## Types of changes
+
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update (Documentation content changed)
+- [ ] Other (please describe):
+
+## Checklist
+
+_Put an `x` in the boxes that apply_
+
+- [ ] I have read the [CONTRIBUTING](https://github.com/mozilla/fxa/blob/master/CONTRIBUTING.md) guidelines
```

Comment:
I feel as though this would get cumbersome with each PR. Also, if it's your first time opening a PR in the repo GitHub will prompt you to read the Contributing doc.

Reply:
Yeah, but I think it is useful if someone is returning contributor. But if it's not necessary I can remove it.

Reply:
I don't mind this checkbox in particular.

It could also be useful to have a checkbox for, "my commit is GPG signed." Generally that's where I see issues.

Reply:
Thanks I will add gpg and remove contributing one